src/Http/AccessToken.php

```diff
@@ -62,6 +95,19 @@ public function touch()
         return $this->save();
     }
 
+    public function updateLastActivity(ServerRequestInterface $request)
```

Comment:
Mixed feelings about this method name, that's not the only thing we're updating. Maybe we could move this logic into `touch` (in which case ignore my comment below), and make $request nullable temporarily (and required in the next version to avoid the unnecessary BC break)?

Reply:
This is the plan now